pkgs/applications/version-management/git-and-tools/git-when-merged/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, python }:
+
+stdenv.mkDerivation rec {
+  pname = "git-when-merged";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "mhagger";
+    repo = "${pname}";
+    rev = "refs/tags/v${version}";
+    sha256 = "0sw98gmsnd4iki9fx455jga9m80bxvvfgys8i1r2fc7d5whc2qa6";
+  };
+
+  buildInputs = [python];
```

Comment:
(I think we typically use spaces after brackets)

Reply:
fixed with `nixfmt`